LRU cache for decoded chunks (#306)
zarr-developers/zarr-python: docs/tutorial.rst

    b'Hello from the cloud!'
    0.0009490990014455747

The above :class:`zarr.storage.LRUStoreCache` wraps any Zarr storage class, and stores

> encoded chunks. So every time cache is accessed, the chunk has to be decoded. For cases

Reviewer: ```suggestion
encoded chunks. Every time the cache is accessed, the chunk must be decoded. For cases
```
